### PR TITLE
Listing players on Draw page

### LINKED ISSUE #16: Draw - Listing players
Listing players on Draw page

## Problem

- How can I see my friend?
- I want to win, put me top 1
- How many point I have?
- How I know who is drawing?

## Proposal

image

### PR DESCRIPTION
**Pull request by zyhou**
resolve #16

### CODE REVIEW

**Comment by Euregan** [comment]
#### 📝Changed routes:


| [**/draw**](https://oeaki-git-listing-player-draw-page.euregan.now.sh/draw) |
|:-:|
|<a href="https://oeaki-git-listing-player-draw-page.euregan.now.sh/draw">Screenshot of /draw</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/qqDlzOs3rtu0yIgC_t88o9nn0lA_.png">(view full size)</a> |







Commit 1f3cd08696c24a7e3c16e9e95d29ff4a4fce8c36 (https://oeaki-git-listing-player-draw-page.euregan.now.sh).

File: components/ListingPlayer.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import React from 'react';
```
Comment (zyhou) [inline components/ListingPlayer.js:1]: remove this files in other PR

File: pages/draw.js [hunk 2]
```diff
@@ -1,13 +1,41 @@
 import React from 'react';
```
Comment (zyhou) [inline pages/draw.js:1]: remove this files in other PR

**Comment by zyhou** [comment]
Merge https://github.com/Euregan/oeaki/pull/35 before pls :)

File: pages/draw.js [hunk 3]
```diff
@@ -1,13 +1,41 @@
...
+//             let rank = index + 1;
+
+//             const playerAbove = index !== 0 && acc[index - 1];
+//             if (playerAbove) {
+//                 rank = player.points === playerAbove.points ? playerAbove.rank : playerAbove.rank + 1;
+//             }
+
+//             return [...acc, { ...player, rank }];
+//         }, []);
+// };
```
Comment (Euregan) [inline pages/draw.js:29]: ohno

**Pull request merged**